### DISCUSSION TITLE
[Request] Custom Recipes can accept ANY damage varient

### DISCUSSION THREAD

**Discussion by TheWranger** [Zrips/CMI#402] [Feature Requests]
Allow us to configure custom recipes so that they will accept ANY damage variant of a block.
        ==: org.bukkit.inventory.ItemStack
        type: WOOD_STEP
        damage: ANY

Could be used to make something like this, where a single recipe is configured to accept any variant of STONE:
![image](https://user-images.githubusercontent.com/24642838/40278028-cb15c20e-5c6c-11e8-9e72-4f5975136cd5.png)

**Comment by MCTyler** [comment]
https://github.com/Zrips/CMI/issues/351

**Comment by TheWranger** [comment]
@MCTyler No, that isn't what I'm suggesting.
I want to be to make a single recipe for a stone pickaxe that can accept any variant of stone.

**Comment by MCTyler** [comment]
Looks like what @CloudeLecaw is EXACTLY asking for. Irregardless, they are similar requests and need to be linked. Sorry if it offends you to link similar requests.

**Comment by TheWranger** [comment]
CloudeLecaw is asking for the ability to set specific requirements for item recipes (things like item lore, durability, item name, attributes, etc.)

I'm asking for the ability to set up recipes so that they will accept ANY damage variant instead of just 1 like they currently do.
We're asking for two completely different things.

**Comment by MCTyler** [comment]
Variants in Recipes.....Ok.

**Comment by CloudeLecaw** [comment]
yes but you should close this one and add the Request to my old post ;)
Because acepting all variants of Stone can also be called "an abillity" ;-)
So please post this post aggain (or link this one) in #351 
to  add your request there :)

other Thread that handle about this theme allready (changes in custom recipes)
https://github.com/Zrips/CMI/issues/351
https://github.com/Zrips/CMI/issues/300
https://github.com/Zrips/CMI/issues/90

if each one open a new request for custom recipes idea before even one is done..then we have 100 threads after a whil ewith one theme :)

thanks

**Comment by MCTyler** [comment]
No need to close, when you link them they all get combined....
